client/src/components/RelatedItems/OutfitCard.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const OutfitCard = ({ outfit }) => {
+  const [product, setProduct] = useState(outfit);
+  const [rating, setRating] = useState();
+  const [price, setPrice] = useState(null);
+  const [imgURL, setImgURL] = useState();
+
+  // Need to abstract this function or Scott's function to App
+  // need to determine how to round
+  const calcRating = (reviews) => {
+    let totalStars = 0;
+
+    for (let i = 0; i < reviews.length; i++) {
+      totalStars += reviews[i].rating;
+    }
+    let rating = totalStars / reviews.length;
+    let floor = Math.floor(rating);
+    let decimal = rating - floor;
+
+    if (decimal <= .25) {
+      setRating(floor);
+    } else if (decimal <= .5) {
+      setRating(floor + .25);
+    } else if (decimal <= .75) {
+      setRating(floor + .5);
+    } else {
+      setRating(floor + .75);
+    }
+  };
+
+  useEffect(() => {
+    axios.get(`/reviews/${product.id}`)
+      .then(result => {
+        let reviews = result.data.results;
+        calcRating(reviews);
+      });
+
+    axios.get(`/products/${product.id}/styles`)
+      .then(result => {
+        let styles = result.data.results;
+
+        for (let i = 0; i < styles.length; i++) {
+          if (styles[i]['default?'] === true) {
```

Comment:
what if there's a sale price? do the cards only show the original price?

Reply:
I totally did not account for that yet. Right now they only show the original price or the default price if no default was ever set. Lemme make that change real quick